### DISCUSSION TITLE
Allow to edit the CSV separator in SQL Request export

### DISCUSSION THREAD

**Discussion by PrestaEdit** [PrestaShop/PrestaShop#35866] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

As now, the SQL Request export use by default the `;` char.

Regarding : https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Export/FileWriter/ExportCsvFileWriter.php#L58

### Expected behavior

Add an option (with a legacy HelperOptions, 😎) to allow to use `;`or `,` separator.

### Steps to reproduce

Export a SQL Request.

### PrestaShop version(s) where the bug happened

8.1.x

### Your company or customer's name goes here (if applicable).

@Wepika

**Comment by Kaikina** [comment]
Willing to PR this

**Comment by AureRita** [comment]
Hi @PrestaEdit 

Thank you for your report, Currently your issue seems more like a feature, are you agree with that ? 

In this case, we can transform this into a discussion with the PM to priorize this feature. 

Waiting for feedback

**Comment by PrestaEdit** [comment]
Hi,

It's on you to decide. It could be a feature, cause it's a change.
But it's like a bug could the char used is in a format instead of another, and could be an issue while needed.

I let you decide how to define this one, no worries about the choice that will be made.

**Comment by AureRita** [comment]
ok, I'll transform this issue into a discusion and in a feature. Thank you 

ping @PrestaShop/product-council
